docs-src/typescript/how-to-send-a-signal-with-start-in-typescript.md

```diff
@@ -13,7 +13,7 @@ tags:
 
 ```typescript
 import {WorkflowClient} from "@temporalio/client";
-import {yourWorkflow, joinSignal} from "./workflows";
+import {joinSignal, yourWorkflow} from "./workflows";
```

Comment:
One of the primary purposes of #1596 was to add spaces around the curly braces in TS import statements, which lines up with how the TS SDK docs' style. So this should be `import { joinSignal, yourWorkflow } from "./workflows'";`